python/rascaline/examples/le-basis.py

```diff
@@ -0,0 +1,203 @@
+"""
+.. _example-le-basis:
+
+Laplacian eigenstate basis
+==========================
+
+.. start-body
+"""
+
+# %%
+#
+# This script illustrates how to generate a spherical expansion using
+# the Laplacian eigenstate basis (https://doi.org/10.1063/5.0124363),
+# both using truncation with l_max, n_max hyper-parameters and with an
+# eigenvalue threshold.
+
+import ase.io
+import numpy as np
+import scipy.optimize
+from scipy.special import jv
+
+from metatensor import Labels, TensorBlock, TensorMap
+import rascaline
+
+try:
+    import torch
+    torch.set_default_dtype(torch.float64)
+    import torch_spex
+    HAS_TORCH_SPEX = True
+except ImportError:
+    HAS_TORCH_SPEX = False
+
+# %%
+#
+# First using a truncation with l_max, n_max hyper-parameters (easy):
+cutoff = 4.4
+l_max = 4
+n_max = 6
+
+structures = ase.io.read("dataset.xyz", ":10")
+
+spliner = rascaline.utils.SoapSpliner(
+    cutoff=cutoff,
+    max_radial=n_max,
+    max_angular=l_max,
+    basis=rascaline.utils.SphericalBesselBasis(cutoff=cutoff, max_radial=n_max, max_angular=l_max),
+    density=rascaline.utils.DeltaDensity(),
+    accuracy=1e-8,
+)
+
+calculator = rascaline.SphericalExpansion(
+    cutoff=cutoff,
+    max_radial=n_max,
+    max_angular=l_max,
+    center_atom_weight=1.0,
+    radial_basis=spliner.compute(),
+    atomic_gaussian_width=-1.0,  # will not be used due to the delta density above
+    cutoff_function={"ShiftedCosine": {"width": 0.5}},
+)
+
+spherical_expansion = calculator.compute(structures)
+
+# %%
+#
+# Now we will calculate the same basis with an eigenvalue threshold
+# (more involved), which affords a better accuracy/cost ratio, using
+# property selection.
```

Comment:
The difference is not very clear here; could you maybe add a graph (ascii art or matplotlib) explaining the square/non square basis and why this matter?